pkg/security/apparmor/shim.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apparmor
+
+import "github.com/opencontainers/runc/libcontainer/apparmor"
+
+// Shim interface allows injection of bad host configuration for testing purposes
+type Shim interface {
+	IsEnabled() bool
+}
+
+type shim struct{}
+
+func (s shim) IsEnabled() bool {
+	return apparmor.IsEnabled()
+}
```

Comment:
Do we really need to put this code into a separate file?

Reply:
Not really - moved it into `validate.go` for simplicity.